IDE/GCC-ARM/Header/user_settings.h

```diff
@@ -171,16 +171,15 @@ extern "C" {
     #undef  ECC_TIMING_RESISTANT
     #define ECC_TIMING_RESISTANT
 
-    /* Enable cofactor support */
     #ifdef HAVE_FIPS
         #undef  HAVE_ECC_CDH
-        #define HAVE_ECC_CDH
-    #endif
+        #define HAVE_ECC_CDH /* Enable cofactor support */
+
+        #undef NO_STRICT_ECDSA_LEN
```

Comment:
In addition wouldn't it make sense to add something like this at the bottom of `wolfssl/wolfcrypt/settings.h`?

```
/* For FIPS make sure the ECDSA encoding allows extra bytes */
#if !defined(NO_STRICT_ECDSA_LEN) && defined(HAVE_FIPS) && \
        defined(HAVE_FIPS_VERSION) && HAVE_FIPS_VERSION >= 2    
    #define NO_STRICT_ECDSA_LEN
#endif
```